src/e2e/app.ts

```diff
@@ -796,12 +796,12 @@ export class App {
   }
 
   /**
-   * Trigger a download and wait for it to complete.
+   * Trigger a hex ifle save and wait for the download to complete.
```

Comment:
```suggestion
   * Trigger a hex file save and wait for the download to complete.
```

Reply:
`ifle` feels like it could be a Python keyword 😄